README.md

```diff
@@ -12,28 +20,50 @@ Install the dependencies, compile the code and build the docker
 make install
 ```
 
+### Install prod
+
+Install the dependencies, compile the code and build the docker in production mode
+
+``` bash
+make install-prod
+```
+
+### Publish
+
+Publish the docker in the docker hub [15-puzzle-api](https://hub.docker.com/r/luwangel/15-puzzle-api/).
+_Note: be careful, you should be logged before!_
+
+``` bash
+make publish
+```
+
+## Run the project
+
 ### Run the game
 
-Run the 15-puzzle game
+Run the 15-puzzle game in dev or prod mode
 
 ``` bash
 make run
+make run-prod
 ```
 
-Note: you can use en environment variable `SIZE` to define the puzzle size
+_Note: you can use en environment variable `SIZE` to define the puzzle size_
 
 For example with a size equals to 5:
 
 ``` bash
 SIZE=5 make run
+SIZE=5 make run-prod
```

Comment:
Use environment variable there ?

Reply:
Yes, because there is no options to change the puzzle size inside the game.